ament_clang_tidy/ament_clang_tidy/main.py

```diff
@@ -77,6 +77,9 @@ def main(argv=sys.argv[1:]):
         '--system-headers',
         action='store_true',
         help='Displays errors from all system headers')
+    parser.add_argument(
+        '--packages-select', nargs='*', metavar='PKG_NAME',
+        help='Only process a subset of packages')
```

Comment:
This was copied directly from the `colcon` source for the argument of the same name.  I wanted to achieve the same sort of behavior as there is in colcon for selecting packages.